README.md

```diff
@@ -19,35 +19,38 @@
 
 </div>
 
-# Here is our website to collect data:
-
-[open-assistant.io](https://open-assistant.io)
-
-(project documentation lives [here](https://laion-ai.github.io/Open-Assistant/))
-
 # Table of Contents
 
 - [What is Open Assistant?](#what-is-open-assistant)
-- [Do you want to try it out?](#do-you-want-to-try-it-out)
-- [The Plan](#the-plan)
+- [Useful Links](#useful-links)
+- [How To Try It Out?](#how-to-try-it-out)
 - [The Vision](#the-vision)
-- [How can you help?](#how-can-you-help)
-- [I’m in! How do I contribute?](CONTRIBUTING.md)
+- [The Plan](#the-plan)
+- [How You Can Help](#how-you-can-help)
 
 ---
 
 ## What is Open Assistant?
 
-<p align="center">
-    Open Assistant is a project meant to give everyone access to a great chat based large language model.
-</p>
+Open Assistant is a project meant to give everyone access to a great chat based
+large language model.
 
 We believe that by doing this we will create a revolution in innovation in
 language. In the same way that stable-diffusion helped the world make art and
 images in new ways we hope Open Assistant can help improve the world by
 improving language itself.
 
-## Do you want to try it out?
+# Useful Links
+
+- Project Repository -
+  [LAION-AI/Open-Assistant](https://github.com/LAION-AI/Open-Assistant)
+
+- Data Collection - [open-assistant.io](https://open-assistant.io)
+
+- Project Documentation -
+  [projects.laion.ai/Open-Assistant](https://projects.laion.ai/Open-Assistant/)
+
+## How To Try It Out?
```

Comment:
With this rewording it's no longer a question so we should remove the question mark

Reply:
Ah yes, thank you for pointing that out.